layers/meta-balena-beaglebone/recipes-support/resin-init/resin-init-flasher.bbappend

```diff
@@ -3,9 +3,17 @@ INTERNAL_DEVICE_KERNEL:beaglebone = "mmcblk1"
 # Except on the beaglebone-ai64
 INTERNAL_DEVICE_KERNEL:beaglebone-ai64 = "mmcblk0"
 
+# Because eMMC enumeration on the beagpleplay is different from other Beaglebone devices (?)
+INTERNAL_DEVICE_KERNEL:beagleplay = "mmcblk0"
+INTERNAL_DEVICE_KERNEL:beagleplay-k3r5 = "mmcblk0"
+
 # the name of the bootloader config file to be written to internal media
 INTERNAL_DEVICE_BOOTLOADER_CONFIG:beaglebone = "uEnv.txt_internal"
+INTERNAL_DEVICE_BOOTLOADER_CONFIG:beagleplay = "uEnv.txt_internal"
+INTERNAL_DEVICE_BOOTLOADER_CONFIG:beagleplay-k3r5 = "uEnv.txt_internal"
 
 # the pathname (relative to /boot/) where the internal bootloader config file should reside in
 # (currently assuming this is on partition 1 of the internal media)
 INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH:beaglebone = "/uEnv.txt"
+INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH:beagleplay = "/uEnv.txt"
+INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH:beagleplay-k3r5 = "/uEnv.txt"
```

Comment:
We do not care about the k3r5 machine so we can remove this

Reply:
OK.

Reply:
Done.